### ISSUE TITLE
Invalid `aria-hidden` prop in `Help`

### ISSUE THREAD

**Issue by Parsium** [Sage/carbon#4698]
### Current behaviour

A failed prop type warning is raised with `Help` - on line 99 in the component's code, `aria-hidden` is supplied with a type of string and not the expected boolean.

### Expected behaviour

No warning is raised.

### CodeSandbox or Storybook URL

https://codesandbox.io/s/hungry-rhodes-jo8r8

### Suggested Solution

Change the type of the supplied value to `aria-hidden` from string to boolean

### Carbon Version

102.7.3

### What browsers are you seeing the problem on?

Chrome

### What Operating System are you seeing the problem on?

MacOS

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by edleeks87** [comment]
I think we should set the type to be `oneOf(["true", "false", undefined])` 

https://www.w3.org/TR/wai-aria-1.1/#aria-hidden

**Comment by grabkowski** [comment]
this issue is already resolved in https://github.com/Sage/carbon/pull/4630.
